guide/svelte.md

```diff
@@ -102,6 +102,8 @@ And finally, import `windi.css` in your main layout:
 
 <slot></slot>
 ```
+> If you want to customize tailwind config, make sure that the config file extesnions is `cjs` e.g. `tailwind.config.cjs`
```

Comment:
I do not think that is true.
If your svelte config is in `.cjs` it might be necessary to use `.cjs` for the config as well. But I think svelte did an update where they also support esm, didn't they?

Reply:
@alexanderniebuhr you might be right on this one, I am not sure about the update that svelte made, I haven't seen it in their website and I am using `"1.0.0-next.75"`. yes the svelte config is in `.cjs` and when I use `.js` with tailwind config I get the following error: 
```
$ svelte-kit dev --open
[vite-plugin-windicss] Error on loading configurations
ReferenceError: module is not defined
    at ...../tailwind.config.js:2:1
```

do you think we should update it to mention that if it's used with `.cjs` svelte config ?

Reply:
@alexanderniebuhr I agree with you, wording could be confusing. How about if it's changed to: 

```
If your sveltekit config is using `.cjs` extension, and you have a tailwind config file then it needs to follow the same extension type e.g. `tailwind.config.cjs`. 
```

Reply:
yes I like that change, maybe we can add a note that this is caused by cjs and esm, but do not worry we are working on the next docs, so we will make sure wording is good there